local-tests/chainrunner/node.py

```diff
@@ -24,13 +24,19 @@ def __init__(self, binary, chainspec, path, logdir=None):
         self.process = None
         self.flags = {}
         self.running = False
+        self.log_level = {}
```

Comment:
Couldn't you just add the logging info to `flags`? Seems to me like now there are two incompatible ways of doing the same thing and no logic to disambiguate in case of conflict...

Reply:
well, the debug flags have the form of `-ltarget=level`, so it neither matches neither value nor non-value flags that we are using

Reply:
Could be done with the long form `--log target=level`